examples/log_files.yml.example.advanced

```diff
@@ -5,7 +5,8 @@ files:
   - /opt/misc/*.log
   - /home/**/*.log
   - /var/log/mysqld.log
-  - /var/run/mysqld/mysqld-slow.log
+  - path: /var/run/mysqld/mysqld-slow.log
+    tag: slow_queries.log
```

Comment:
Great suggestions! @markdascher came up with an alternate example that I adapted that might be clearer.